UglyTicTacToe/Models/GameLogic.cs

```diff
@@ -0,0 +1,74 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace UglyTicTacToe.Models
+{
+    public class GameLogic
```

Comment:
If the `GameLogic` class isn't designed to be instantiated, then add the `static` keyword to make that intention clearer.